src/Installer/dnup/ArchiveDotnetInstaller.cs

```diff
@@ -421,12 +422,16 @@ public void Dispose()
         }
     }
 
+    //  TODO: InstallerOrchestratorSingleton also checks existing installs via the manifest.  Which should we use and where?
```

Comment:
I think we must verify both as the manifest could be wrong if the user deletes it themselves.